### PR TITLE
Bootstrap 5: Dropdowns

### PR DESCRIPTION
**Pull request by jovyntls**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [x] Code maintenance
- [ ] Others, please explain:

Part of #1702 

**Overview of changes:**
Migrate the breaking changes to dropdowns from Bootstrap 4->5. 
Dropdowns require Popper.js which is not included in `bootstrap.min.js`, hence `bootstrap.bundle.min.js` is required. The minified bootstrap files have been updated to use `bootstrap.bundle.min.js`, though the name is unchanged to match the previous name (`bootstrap-utility.min.js`).

Quick summary of the rationale for some changes:
* `data-bs-toggle` is needed to correctly position dropdowns
* Some keydown events are ignored in Bootstrap inputs, hence `keyup` is used instead

**Anything you'd like to highlight / discuss:**
N/A

**Testing instructions:**
Components using the Bootstrap dropdown component should be functional with no UI changes. These include the SearchBar component and Dropdown component.

**Proposed commit message: (wrap lines at 72 characters)**
```
Migrate dropdown components
```

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: packages/vue-components/src/Submenu.vue [hunk 1]
```diff
@@ -2,13 +2,14 @@
   <li
     ref="submenu"
     :class="[addClass, 'dropdown-submenu',
-             { 'dropright': dropright, 'dropleft': dropleft }]"
+             { 'dropend': dropright, 'dropstart': dropleft }]"
```
Comment (jonahtanjz) [inline packages/vue-components/src/Submenu.vue:5]: Will need to adjust the CSS of the submenu. Need to remove the extra left padding and change the colour of the text.

image

File: packages/vue-components/src/Submenu.vue [hunk 2]
```diff
@@ -97,7 +98,7 @@ export default {
     $el.onBlur(() => { this.hideSubmenu(); }, false);
```
Comment (jonahtanjz) [inline packages/vue-components/src/Submenu.vue:98]: I think the `onBlur` is not fully working properly? 

Master branch submenu (Notice how the `submenu`s will close when they are blurred.):

https://user-images.githubusercontent.com/47470981/162556008-f5726996-d928-4302-b84d-3ac42e9e3cd5.mp4


Current branch submenu (Multiple `submenu`s of the same dropdown can be opened, resulting the overlapping `submenu`s):

https://user-images.githubusercontent.com/47470981/162556078-f672eed3-cefd-40d3-978e-13fa0a053871.mp4

File: packages/vue-components/src/Searchbar.vue [hunk 3]
```diff
@@ -179,7 +180,8 @@ export default {
         'dropdown-menu',
         'search-dropdown-menu',
         { show: this.showDropdown },
-        { 'dropdown-menu-right': this.menuAlignRight },
+        { 'd-none': !this.showDropdown },
+        { 'dropdown-menu-end': this.menuAlignRight },
```
Comment (jonahtanjz) [inline packages/vue-components/src/Searchbar.vue:184]: The dropdown for the searchbar is also a little buggy. The first time is fine, aligned to the right but on the second time onward, the dropdown is aligned to the left.

https://user-images.githubusercontent.com/47470981/162556182-e312094f-7e50-42df-a51b-c542f4618955.mp4

**Review by jonahtanjz — COMMENTED** [review]
Thanks @jovyntls 👍 

Found a couple of issues with the `Submenu` and `Searchbar`

File: packages/vue-components/src/Submenu.vue (hunk 2 above)
Comment (jovyntls) [reply packages/vue-components/src/Submenu.vue:98]: Thank you for the tests @jonahtanjz !

**Comment by jovyntls** [comment]
Thank you so much for testing @jonahtanjz ! I've pushed some fixes :)

**Review by jonahtanjz — APPROVED** [review]
LGTM 👍

**Pull request merged**